### PR TITLE
Trim trailing whitespace

### PR DESCRIPTION
**Pull request by bennettgoble**
Start trimming trailing whitespace, but limit the blast radius to a handful of file types. Also added a .git-blame-ignore-revs file so that whitespace changes are excluded from git blame.

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Replace tabs with spaces
+1b68f71348ecf3983b76b40d7940da8377f049b7
```
Comment (marchcat) [inline .git-blame-ignore-revs:2]: 👍

**Review by marchcat — APPROVED** [review]
Thank you!

**Comment by bennettgoble** [comment]
Hey, @marchcat, which branch should this get merged into?

**Comment by marchcat** [comment]
@bennettgoble, let's merge this to [DRTVWR-600-maint-A](https://github.com/secondlife/viewer/tree/DRTVWR-600-maint-A), I'll add more commits to the blame ignore list later.

**Pull request merged**